src/js/angular/repositories/templates/ontop-repo.html

```diff
@@ -20,40 +20,42 @@ <h4>Ontop configuration</h4>
     </div>
     <div ng-repeat="labelName in propertiesFileParams" class="form-group row indented-div"
          ng-if="selectedDriver.driverType !== genericDriverType">
-        <label for="{{labelName}}" class="col-xs-3 col-sm-3 col-md-2 col-form-label">
+        <label for="{{labelName}}" class="col-xs-12 col-md-3 col-lg-2 col-form-label">
             {{supportedDriverLabels[labelName] + (isRequiredField(labelName) ? '*' : '')}}</label>
-        <div class="col-xs-4 col-sm-4 col-md-4" tooltip="{{repoTooltips.ontop[labelName]}}">
+        <div ng-class="{'col-xs-8 col-md-6 col-lg-5': labelName === 'url', 'col-xs-12 col-md-6 col-lg-5': labelName !== 'url'}"
+             tooltip="{{repoTooltips.ontop[labelName]}}">
             <input id="{{labelName}}" class="form-control" ng-readonly="isReadOnly(labelName)"
-                   type="{{getInputType(labelName)}}"
+                   type="{{getInputType(labelName)}}" min="1" max="65535"
                    ng-required="{{isRequiredField(labelName)}}"
                    placeholder="{{isRequiredField(labelName) ? 'This field is required' : ''}}"
-                   ng-model="selectedDriver.jdbc[labelName]" ng-blur="concatURL(labelName)">
+                   ng-model="selectedDriver.jdbc[labelName]" ng-change="concatURL(labelName)">
         </div>
-        <div class="form-group col-xs-2 col-sm-2 col-md-2">
-            <div class="wrapper-div" tooltip="{{repoTooltips.ontop.testConn}}">
-                <button id="testConnection1" ng-click="validateOntopPropertiesConnection()"
-                        ng-disabled="!selectedDriver.jdbc.hostName || !selectedDriver.jdbc.userName || !selectedDriver.jdbc.databaseName"
-                        ng-if="labelName === 'url'"
-                        type="submit"
-                        class="btn btn-primary">Test connection
-                </button>
-            </div>
+        <div class="col-xs-12 col-md-3" ng-if="labelName === 'driverClass'">
             <em class="icon-warning icon-2x"
-                ng-if="labelName === 'driverClass' && !classAvailable"
+                ng-if="!classAvailable"
                 tooltip="{{repoTooltips.ontop.url}}"
```

Comment:
This is the wrong tooltip I think. It is about the JDBC url.

Reply:
The proper tooltip is repoTooltips.ontop.driverClassWarning.

Reply:
Fixed.